cli/borges/consumer.go

```diff
@@ -53,9 +56,16 @@ func (c *consumerCmd) Execute(args []string) error {
 	wp.SetWorkerCount(c.WorkersCount)
 
 	ac := borges.NewConsumer(q, wp)
-	ac.Start()
 
-	return nil
+	var term = make(chan os.Signal)
+	go func() {
+		_ = <-term
+		log.Info("signal received, stopping...")
+		ac.Stop()
+	}()
+	signal.Notify(term, syscall.SIGTERM, syscall.SIGINT)
```

Comment:
Maybe `signal.Notify(term, os.Interrupt, os.Kill)` so we can avoid import `syscall`

Reply:
We also need to handle `syscall.SIGTERM`, so we will import it anyway

Reply:
Done